Server/controllers/authController.js

```diff
@@ -69,25 +90,22 @@ const loginController = async (req, res) => {
     }
     // Compare password
     const match = await user.comparePassword(req.body.password);
-    console.log(user);
     if (!match) {
       throw new Error("Password does not match!");
     }
 
-    // Copy user to remove password.  More memory, but better than mutating user?
-    // We can move this to the DB layer if we want
-    // Probably not neccessary at all as we'll likely return a JWT instead of a user
+    // Remove password from user object.  Should this be abstracted to DB layer?
     const userWithoutPassword = { ...user._doc };
     delete userWithoutPassword.password;
 
```

Comment:
This approach looks fine right now. Let's just keep it that way, later we can refactor it if we want.